api/v1/inferencepool_types.go

```diff
@@ -165,6 +170,10 @@ type PoolStatus struct {
 	// +kubebuilder:validation:MaxItems=8
 	// +kubebuilder:default={{type: "Accepted", status: "Unknown", reason:"Pending", message:"Waiting for controller", lastTransitionTime: "1970-01-01T00:00:00Z"}}
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
+
+	// GatewayRef indicates the gateway that observed state of InferencePool.
+	// +required
```

Comment:
Should it be required? I thought InferencePool can exist without Gateway? cc @robscott @kfswain @danehans @nirrozenbaum

Reply:
It was already required (as you can see there's no change on the generated CRD)